test/tests/materials/stateful_prop/many_stateful_props.i

```diff
@@ -0,0 +1,129 @@
+# This test creates several unused stateful properties.
+# It's here to make sure that we don't consume too much
+# memory if we store them all. With 180x180 elements
+# we were previously seeing nearly a Gigabyte of memory
+# consumed using TBB's map. We are now using unordered
+# map which saves us 6x to 8x on memory.
```

Comment:
Maybe, However I think our test files should be much more descriptive of what they are testing. Users/Developers should be able to access this information without diving into commit messages.